src/Fraction.jl

```diff
@@ -29,12 +29,6 @@ function characteristic(R::FracField{T}) where T <: RingElem
    return characteristic(base_ring(R))
 end
 
-function check_parent(a::FracElem, b::FracElem, throw::Bool = true)
```

Comment:
this method (and many more of the removed ones) have three arguments. However, your newly added one has two args and one kwarg. Thus, all callsites that specify the `throw` argument break.

Reply:
Argh, that was not intended. Fixed the "generic" method